src/anndata/_core/merge.py

```diff
@@ -939,21 +939,34 @@ def gen_outer_reindexers(els, shapes, new_index: pd.Index, *, axis=0):
     return reindexers
 
 
+def missing_element(
+    n: int,
+    els: list[SpArray | sparse.csr_matrix | sparse.csc_matrix | np.ndarray | DaskArray],
+    axis: Literal[0, 1] = 0,
+    fill_value: Any | None = None,
+) -> np.ndarray | DaskArray:
+    """Generates value to use when there is a missing element."""
+    should_return_dask = any(isinstance(el, DaskArray) for el in els)
+    non_missing_elem = next(el for el in els if not_missing(el))
+    off_axis_size = 0 if not should_return_dask else non_missing_elem.shape[axis - 1]
```

Comment:
could you add a comment why it’s `0 if not should_return_dask`?

I assume that’s because of broadcasting?

Reply:
Yes, I assume to but I didn't write this originally so I just replicated what was there before.  I understand in principle what purpose a `(0,n)` sized numpy array is but not practically so I didn't look into it

Reply:
ideal point in time to figure it out! does broadcasting work with the dask array as well?

Reply:
OR rather it works, but is not performant